test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -142,9 +144,11 @@ call_log_manager::call_log_commit_transaction(const json &call_log_entry)
 
     int ret = session->commit_transaction(config);
 
+#ifdef DEBUG
```

Comment:
WT-10944 is [fixing this](https://github.com/wiredtiger/wiredtiger/pull/9051) with `[[maybe_unused]]` annotations rather than `DEBUG #ifdef`s. We can remove the changes to `call_log_manager.c` once we get back to this PR post WT-10740

Reply:
My bad @mickgraham-mongodb. I forgot that this PR was handing this issue. Do you want me to merge WT-10944 or I can close my PR? Let me know.

Reply:
@Siddhartha8899 please merge WT-10944. I can update this PR after you have completed and post WT-10740 being addressed.